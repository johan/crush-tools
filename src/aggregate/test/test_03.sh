test_number=03
description="user-specified agg-field labels"

outfile="$test_dir/test_$test_number.actual"
expected="$test_dir/test_$test_number.expected"

$bin -p -k 1 -s 3 -l Sum-Field "$test_dir/test.in" > "$outfile"

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
