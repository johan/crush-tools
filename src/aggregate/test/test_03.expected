Text-1	Sum-Field
first text value	3
second text value	8
